chia/cmds/farm_funcs.py

```diff
@@ -232,6 +233,10 @@ async def summary(rpc_port: int, wallet_rpc_port: int, harvester_rpc_port: int,
     minutes = -1
     if blockchain_state is not None and plots is not None:
```

Comment:
This whole time computation should probably be in the if-clause 5 lines above, not it a second thing that repeats the checks. `plots` is actually not even used in the computation.

Reply:
this seems like it should be a separate pr